examples/typed-component/sample_component.yaml

```diff
@@ -5,6 +5,8 @@ metadata:
 spec:
   workload:
     type: web-service
+    apiVersion: core.oam.dev/v1alpha2
```

Comment:
`type` is mutually exclusive with `apiVersion` and `kind`, so please leave this yaml as is. Typed component is a experimental design in oam-runtime and it's not compliance with OAM spec v0.2.1.

Also, we decide not to accept this format in OAM spec per recent community meeting, so `type` will be deprecated soon.

Reply:
If type is deprecated we should remove it though? That’s the opposite. So don’t know what to do.

Reply:
Yes, exactly, it should be removed. That's why I suggest leave it as is unless you want to take this burden (I don't think so, and it's safer to do clean up in separate PR).